Nrf profiler timestamp (#28314)
nrfconnect/sdk-nrf: doc/nrf/libraries/others/nrf_profiler.rst

> Since Application Event Manager events are converted to nRF Profiler events by the :ref:`app_event_manager_profiler_tracer`, you can configure the nRF Profiler to profile custom events or Application Event Manager events, or both.

Reviewer: Nit: commit message is a bit misleading - `CONFIG_SYS_CLOCK_TICKS_PER_SEC` is something else

Reviewer: Not addressed

Reviewer: Still not addressed

.. note::

>    Starting from |NCS| v3.4.0, the nRF Profiler library introduces a modified string layout for the Info RTT channel to support the nRF54L SoC series.

Reviewer 2: ```suggestion
   Starting from the |NCS| v3.4.0, the nRF Profiler library introduces a modified string layout for the Info RTT channel to support the nRF54L Series SoCs.
```

>    The library now encapsulates system configuration data, such as :kconfig:option:`CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC`, within specific markers to allow the host tools to perform accurate time unit conversions.

Reviewer: ```suggestion
   The library now provides also system configuration data, such as :kconfig:option:`CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC`, within specific markers to allow the host tools to perform accurate time unit conversions for nRF54L Series SoCs.
```

   This modification breaks backward compatibility with older host scripts.

>    Ensure that the host tools version matches the nRF Profiler library version running on the SoC.

Reviewer: ```suggestion
   To ensure proper behavior, use both library and host tools from the same NCS release.
```

Reviewer: (we actually do not version nRF Profiler library and host tools separately)

>    Users must ensure that the host tools version matches the nRF Profiler library version running on the SoC.

Reviewer 2: ```suggestion
.. note::
   Starting from |NCS| v3.4.0, the nRF Profiler library introduces a modified string layout for the Info RTT channel to support the nRF54L SoC series.
   The library now encapsulates system configuration data, such as kconfig:option:`CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC`, within specific markers to allow the host tools to perform accurate time unit conversions.
   This modification breaks backward compatibility with older host scripts.
   Ensure that the host tools version matches the nRF Profiler library version running on the SoC.
```

>    To ensure proper behavior, use both library and host tools from the same NCS release.

Reviewer 2: ```suggestion
   This modification breaks the backward compatibility with older host scripts.
   To ensure proper behavior, use both the library and host tools from the same |NCS| release.
```
